### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by lulucikyo**
Cleaned up the python code
Deleted old training files and logs
Moved unused data files

### CODE REVIEW

File: replication_chexnet_cuda_local.py [hunk 1]
```diff
@@ -177,16 +184,16 @@ def train_model(model, train_loader, val_loader, n_epochs, logfile):
     log = open(logfile, "a")
     log.write("\n\n\nStarted training, total epoch : {}\n".format(n_epochs))
     log.write("Training data size: {}\n".format(len(train_loader)))
-    print("Started training, total epoch : {}\n".format(n_epochs))
-    print("Training data size: {}\n".format(len(train_loader)))
+    print("Started training, total epoch : {}".format(n_epochs))
```
Comment (Yekai-Yu) [inline replication_chexnet_cuda_local.py:187]: Might need to keep the \n

File: replication_chexnet_cuda_local.py [hunk 2]
```diff
@@ -177,16 +184,16 @@ def train_model(model, train_loader, val_loader, n_epochs, logfile):
...
+    print("Training data size: {}".format(len(train_loader)))
 
     for epoch in range(n_epochs):
         gc.collect()
         torch.cuda.empty_cache()
         train_loss = 0
         batch = 0
         log.write("\nStarted epoch {}\n".format(epoch+1))
-        print("\nStarted epoch {}\n".format(epoch+1))
+        print("\nStarted epoch {}".format(epoch+1))
```
Comment (Yekai-Yu) [inline replication_chexnet_cuda_local.py:209]: \n at the front can be removed

File: replication_chexnet_cuda_local.py [hunk 3]
```diff
@@ -289,17 +307,20 @@ def eval_model(model, test_loader, logfile, setstr):
...
+If there is a trained model, it can be loaded.
+"""
 model = DenseNet121(N_LABEL).cuda()
-# load trained model if needed
-model.load_state_dict(torch.load("20trained.pth"))
+# model.load_state_dict(torch.load("trained.pth"))
 
 
+""" Initialize the Dataset"""
 train_dataset = XrayDataSet(DATA_PATH, "final_train.txt", train_sampling=False)
```
Comment (Yekai-Yu) [inline replication_chexnet_cuda_local.py]: this `train_sampling=False` could be removed, I believe it was added by @lytinahome for testing purpose

**Comment by Yekai-Yu** [comment]
We can rename the main program file name to `chexnet_cuda_replication.py`

**Comment by Yekai-Yu** [comment]
I also have some log automation in https://github.com/lulucikyo/CheXNet/commit/52f9b65523cd220d7a71aefda0232854da12b36f , we can add this in another PR afterwards

File: replication_chexnet_cuda_local.py (hunk 3 above)
Comment (Yekai-Yu) [reply replication_chexnet_cuda_local.py]: I mean - `XrayDataSet` can be cleaned as well

File: replication_chexnet_cuda_local.py (hunk 2 above)
Comment (lulucikyo) [reply replication_chexnet_cuda_local.py:209]: Kept to make log file clear to read

File: replication_chexnet_cuda_local.py (hunk 3 above)
Comment (lulucikyo) [reply replication_chexnet_cuda_local.py]: cleaned

**Review by Yekai-Yu — APPROVED** [review]

**Pull request merged**